### ISSUE TITLE
There are no low battery warnings Linux Mint 21.1

### ISSUE THREAD

**Issue by joao-pm-santos96** [linuxmint/cinnamon#11504]
```
 * Cinnamon 5.6.7
 * Linux Mint 21.1 Cinnamon
 * Advanced Micro Devices, Inc. [AMD/ATI] Picasso/Raven 2 [Radeon Vega Series / Radeon Vega Mobile Series]
 * 64 bit
 ```

### Issue

I have two laptops from two different brands and I have the same issue. I mainly use them unplugged, but when the battery is low (bellow 20%) there is no notification. This is a huge issue for those of us who use laptops. I have already used dconf editor but the result is the same.

### Steps to reproduce

Just let your battery drain and checks if there is any warning message.

### Expected behaviour

There should be a battery low warning when power level reaches an defined X%.

**Comment by xgirouxb** [comment]
I also agree this would be a great feature, I just upgrade to 21.1 from the last LTS and I'm a bit disappointed this has not been implemented, seems like a pretty basic requirement... I frequently get accidental shutdowns without warning due to low battery.

**Comment by bokeTanuki** [comment]
Besides the notification, an audible alarm (at 10% or user set) would be nice to have too. I've had to connect the charger to my laptop after watching Mint auto shut down on me suddenly while doing something too many times.  And Mint shuts down on 0-1% battery instead of shutting down at 5% or 10%.
